pkg/swagger/swagger.go

```diff
@@ -231,3 +272,15 @@ func removeNamedSchemas(list NameSchemas, remove string) NameSchemas {
 
 	return result
 }
+
+// TODO: once we upgrade to go 1.18 we can use the slices.Contains function
```

Comment:
can we track this tech debt in a story?

Reply:
https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/15533002